views/jira-manual-app-creation.hbs

```diff
@@ -0,0 +1,142 @@
+        </div>
+
+        <div class="field-group">
+          <label for="callback-url">Callback URL</label>
+          <!--TODO: replace the value by actual data-->
+          <input class="text full-width-field" type="text" id="callback-url"
+                 name="callback-url" value="https://github.atlassian.com/github/callback/V1StGXR8_Z5jdHi6B-myT"
+                 readonly
+          />
+        </div>
+
+        <div class="field-group">
+          <label for="setup-url">Setup URL</label>
+          <!--TODO: replace the value by actual data-->
+          <input class="text full-width-field" type="text" id="setup-url"
+                 name="setup-url" value="https://github.atlassian.com/github/setup/V1StGXR8_Z5jdHi6B-myT"
+                 readonly
+          />
+        </div>
+
+        <div class="field-group">
+          <label for="webhook-url">Webhook URL</label>
+          <!--TODO: replace the value by actual data-->
+          <input class="text full-width-field" type="text" id="webhook-url"
+                 name="webhook-url" value="https://github.atlassian.com/github/webhook/V1StGXR8_Z5jdHi6B-myT"
+                 readonly
+          />
+        </div>
+
+        <div class="jiraManualAppCreation__formTitle">Secrets</div>
+        <div class="jiraManualAppCreation__formSubTitle">
+          Paste secrets from your GitHub Enterprise Server configuration.
+        </div>
+
+        <div class="field-group">
+          <label for="client-secret">GitHub client secret</label>
+          <input class="text full-width-field" type="password" required id="client-secret" name="client-secret" />
+        </div>
+
+        <div class="field-group">
+          <label for="webhook-secret">Webhook secret</label>
+          <input class="text full-width-field" type="password" required id="webhook-secret" name="webhook-secret" />
+        </div>
+
+        <div class="field-group">
+          <label>Private key</label>
+          <div class="jiraManualAppCreation__formFileDropArea">
+            <img class="jiraManualAppCreation__formFileIcon" src="/public/assets/file-upload.svg" alt="Upload">
+            <div>
+              <div class="jiraManualAppCreation__formFileMsg">Upload your private key</div>
+              <div class="jiraManualAppCreation__formFileSubMsg">Upload your generated private key from GitHub</div>
+            </div>
+            <div class="aui-button">Upload</div>
+            <input class="jiraManualAppCreation__formFileInput" id="private-key" required type="file">
+          </div>
+          <div class="jiraManualAppCreation__formFileUploaded">
+            <div id="jiraManualAppCreation__uploadedFile"></div>
+            <span class="aui-icon aui-icon-small aui-iconfont-cross" id="jiraManualAppCreation__clearUploadedFile">Remove</span>
+          </div>
+        </div>
+
+        <button class="jiraManualAppCreation_formSubmitBtn aui-button aui-button-primary">Next</button>
```

Comment:
Where is the error component? It's included in the design so we should definitely add it to this pr.

Reply:
Error component has been added

Reply:
Yes, you have added `required` as a attribute in the form but you are not handling the frontend validation if the form is submitted and those fields are empty,

<img width="733" alt="Screen Shot 2022-07-05 at 12 26 24 pm" src="https://user-images.githubusercontent.com/37155488/177237402-f4333ab6-14b3-4bac-8693-dd74294fd84d.png">

There is also this frontend validation.

<img width="700" alt="Screen Shot 2022-07-05 at 12 26 41 pm" src="https://user-images.githubusercontent.com/37155488/177237421-4cc37645-362d-4be5-b47a-4ce273ac0acd.png">